### PR TITLE
[DE] add degrees sign to temperature expansion rule

### PR DESCRIPTION
**Pull request by hannemann**
Sometimes whisper return 20°C and sometimes 20 Grad

### CODE REVIEW

File: sentences/de/_common.yaml [hunk 1]
```diff
@@ -183,7 +183,7 @@ expansion_rules:
   luefter: "<artikel> (Ventilator[en]|Lüfter)"
   alle_luefter: "<alle> (Ventilatoren|Lüfter)"
   brightness: "{brightness} [Prozent|%]"
-  temperature: "{temperature} [Grad] [{temperature_unit}]"
+  temperature: "{temperature} [Grad] [[°]{temperature_unit}]"
```
Comment (mib1185) [inline sentences/de/_common.yaml:186]:
```suggestion
  temperature: "{temperature}[ ][°|Grad][ ][{temperature_unit}]"
```

i think we should adopt the English approach
https://github.com/home-assistant/intents/blob/bfe1eb8feb4a6ed623c4fcdd670fc200f52f23a2/sentences/en/_common.yaml#L304

**Review by mib1185 — CHANGES_REQUESTED** [review]
Hi @hannemann 
thanks for your contribution, just one small comment 👍

File: sentences/de/_common.yaml (hunk 1 above)
Comment (hannemann) [reply sentences/de/_common.yaml]: I agree :+1:

File: tests/de/climate_HassClimateSetTemperature.yaml [hunk 2]
```diff
@@ -2,8 +2,11 @@ language: de
 tests:
   - sentences:
       - Stelle im Wohnzimmer die Temperatur auf 30 Grad Celsius
+      - Stelle im Wohnzimmer die Temperatur auf 30 °C
```
Comment (mib1185) [inline tests/de/climate_HassClimateSetTemperature.yaml:5]:
```suggestion
      - Stelle im Wohnzimmer die Temperatur auf 30 °C
      - Stelle im Wohnzimmer die Temperatur auf 30°C
```

File: tests/de/climate_HassClimateSetTemperature.yaml [hunk 3]
```diff
@@ -2,8 +2,12 @@ language: de
...
+      - Stelle im Wohnzimmer die Temperatur auf 30°C
```
Comment (mib1185) [inline tests/de/climate_HassClimateSetTemperature.yaml:6]:
```suggestion
```

**Review by mib1185 — APPROVED** [review]

**Comment by hannemann** [comment]
My findings are that no matter what i try if the temperature and unit are something like `30°C` without a space in between the slot temperature contains `30°c` instead of just `30`. IMO that's why the tests fail and should not be the case.

I also found that in the hassil library recognize.py line 1050 the `text` property of the `MatchEntity` is returned unchanged but the `value` was cast to an integer .

In the response template the `text` property is used.
Do we have access to that `value` property in the response template?

**Pull request merged**